### ISSUE TITLE
Possible issue with some rules not working? 

### ISSUE THREAD

**Issue by adam-lynch** [sasstools/gulp-sass-lint#77]
See https://github.com/sasstools/sass-lint/issues/1050. It might be an issue with this plugin or maybe I'm configuring it incorrectly. As the other issue shows, `property-units` doesn't seem to work. But now I'm just noticed that if I have the following:

```coffeescript
    'class-name-format': [
        error
        { 'convention': 'hyphenatedbem', 'convention-explanation': 'HELLO' }
    ]
```

Then `HELLO` isn't shown when the rule is triggered.

**Comment by adam-lynch** [comment]
Nevermind, I'm an idiot. It was a problem with my setup.
